docker-compose-postgres.yml

```diff
@@ -0,0 +1,66 @@
+services:
+  postgres:
+    image: postgres:14
+    container_name: plone-postgres
+    ports:
+      - "127.0.0.1:5432:5432"
+    environment:
+      POSTGRES_USER: plone
+      POSTGRES_DB: local
+      POSTGRES_PASSWORD: qweqwe1!
+    volumes:
+      - type: bind
+        source: ./plone-postgres-data/
+        target: /var/lib/postgresql/data
+  plone:
+    image: plone-postgres-local:latest
+    depends_on:
+      - postgres
+    command: ["./wait-for-it.sh", "postgres:5432", "--", "gosu", "plone", "bin/instance", "console", "--no-cache"]
```

Comment:
I'd strongly prefer we use healthcheck than a script for the dependency/startup order if possible. https://stackoverflow.com/questions/31746182/docker-compose-wait-for-container-x-before-starting-y/41854997#41854997

If we add the healthcheck stanza to postgres and on-failure:restart to plone docker should take care of itself without wait-for-it?

Reply:
I've added healthchecks for each of the services and a dependency of volto on plone too. I've added the `restart: on-failure` bit but haven't been able to test it since the apps prefer to stay alive.